tools/config.schema.yaml

```diff
@@ -0,0 +1,855 @@
+$schema: "http://json-schema.org/draft-07/schema#"
+description: RAPIDS configuration schema
+required:
```

Comment:
Add all existent keys so they are mandatory

Reply:
Still missing most of the phone, fitbit and plot keys

Reply:
Missing time_segment, will add it later

Reply:
Fixed